kompari-cli/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "kompari-cli"
+description = "CLI for reporting tool of image differences for snaphost testing."
```

Maybe just:
```suggestion
description = "Snapshot test reporting CLI."
```
You've typoed snapshot as "snaphost" here. Would you be willing to submit that to upstream https://github.com/crate-ci/typos? (This would be a non-critical correction there)

---

Fixed and added into types issue tracker.